### PR TITLE
Fix python wordmark icon

### PR DESCRIPTION
**Pull request by Kartstig**
- Relates to https://github.com/konpa/devicon/issues/93

### CODE REVIEW

**Comment by iamclaytonray** [comment]
Hey @Kartstig - In #93, you mentioned:

> I have gotten the Python icon to work, but it doesn't release certain backgrounds, casing other subsequent icons to be colored yellow.

Is that the case with this fixed version? I'm not sure what you mean here. Thanks!

**Comment by Kartstig** [comment]
Oh it's been a while. Let me look at this again and get back to you

**Comment by Kartstig** [comment]
What I was referring to in the issue was that `master` had an SVG such that it didn't release the colors, so when you changed them, it affected everything else.

e.g. If you added the python icon, followed by another icon, it would color the proceeding icon yellow from the python icon

Here's the version I have provided:
image

I just verified that this new image has resolved the issue by allowing the update of individual paths inside the SVG without breaking the other paths:
image

image

image


One other thing to note about this update is that it changes the alignment of the words. "python" is now below the icon. I think this is more consistent with the other icons, and makes alignment easier on the front-end.

**Comment by Kartstig** [comment]
@iamclaytonray let me know if you need any updates or clarification. Thanks!

**Comment by iamclaytonray** [comment]
@Kartstig - ah, I see! Sorry, been super slammed with work. I'll try to get back to this tonight but if not, tomorrow morning/afternoon. Thanks again!

**Comment by Thomas-Boi** [comment]
Hello there,
Recently, we just introduced an automation pipeline to Devicon which will help us create the icons. Before we can merge your changes in, there are a few things that we need to do first:

1. Merge the current devicon/develop branch into your PR branch. This contains the build script needed for us to build your icons.
2. Read the `CONTRIBUTING.md` for things to do before merging and review our icon standards.
3. Push your changes and/or branches. We will review them once this is done and merge your PR if everything goes well.


Thank you for contributing to this repo. We are sorry that we haven't accept any PR for awhile. Now that we have a build pipeline, we can merge PRs more quickly than before. 

We hope to see your icons in the Devicon repo soon :)

Cheers,
Devicon Team

**Comment by Kartstig** [comment]
@iamclaytonray i'm revisiting my PRs here. any chance we can get this merged?

**Comment by amacado** [comment]
Let's wait for https://github.com/devicons/devicon/issues/93#issuecomment-755555820 so we get the feedback if the issue is fixed.

File: icons/python/python-original-wordmark.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...58.299,98.261,58.493,98.633z"/> <path fill-rule="evenodd" clip-rule="evenodd" fill="#FEFEFE" d="M58.542,9.978c-0.147,2.503-1.303,3.982-3.691,3.913 c-2.173-0.063-3.363-1.461-3.36-3.646c0.003-2.355,1.24-3.803,3.634-3.74C57.313,6.564,58.374,8.005,58.542,9.978z"/> <path fill-rule="evenodd" clip-rule="evenodd" fill="#FEFEFE" d="M80.299,72.68c-0.374,2.203-1.51,3.646-3.839,3.822 c-2.801,0.212-3.313-2.049-3.328-3.886c-0.013-1.728,0.655-3.663,3.243-3.602C78.863,69.076,79.829,70.538,80.299,72.68z"/> <path fill-rule="evenodd" clip-rule="evenodd" fill="#FEFEFE" d="M12.292,106.926c0-1.001,0.086-2.012-0.018-3.002 c-0.244-2.348,0.134-4.248,2.928-4.644c2.829-0.399,5.252-0.024,6.301,3.182c1.182,3.608,0.921,7.126-0.635,10.537 c-0.926,2.034-2.509,3.016-4.819,2.471c-2.335-0.552-4.314-1.316-3.794-4.417C12.479,109.711,12.292,108.305,12.292,106.926z"/> <path fill-rule="evenodd" clip-rule="evenodd" fill="#FEFEFE" d="M96.401,107.477c-0.115,1.537-0.078,2.688-0.31,3.785 c-0.509,2.423-1.979,4.056-4.47,4.472c-2.529,0.423-3.989-1.104-4.936-3.188c-1.472-3.236-1.562-6.547-0.201-9.843 c0.866-2.1,2.161-3.617,4.756-3.504c2.572,0.113,3.833,1.782,4.525,3.856C96.272,104.566,96.243,106.261,96.401,107.477z"/> </g> </svg>
```
Comment (amacado) [inline icons/python/python-original-wordmark.svg:1]: `enable-background="new 0 0 128 128"` is [depreceated](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/enable-background).

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="128px" height="128px" viewBox="0 0 128 128"> <g> <path fill-rule="evenodd" clip-rule="evenodd" fill="#386F9F" d="M51.386,21.507c3.894,0,7.789,0.061,11.678-0.041 c1.031-0.027,2.882,0.826,2.8-1.318c-0.064-1.745-1.683-1.114-2.663-1.129c-4.393-0.07-8.801-0.236-13.177,0.038 c-3.113,0.194-4.031-0.988-3.64-3.843c0.134-0.985,0.005-2.005,0.026-3.009c0.137-6.684,1.752-8.958,8.235-10.606 c7.141-1.815,14.45-1.755,21.572-0.192c6.402,1.406,9.062,6.387,8.738,10.995c-0.393,5.623-0.089,11.294-0.094,16.945 c-0.009,7.105-3.68,10.876-10.844,10.975c-5.9,0.081-11.807,0.144-17.703-0.019c-7.028-0.196-12.182,5.641-12.218,12.255 c-0.001,0.502,0.086,1.021-0.014,1.505c-0.506,2.429,1.516,6.355-1.059,7.077c-3.919,1.101-8.509,1.101-11.965-2.396 c-1.171-1.181-2.159-2.506-2.679-4.036c-2.778-8.155-3.217-16.413-0.602-24.696c1.63-5.169,6.064-8.384,11.556-8.477 c4.016-0.067,8.034-0.013,12.051-0.013C51.386,21.515,51.386,21.511,51.386,21.507z M58.542,9.978 c-0.168-1.973-1.229-3.414-3.417-3.473c-2.394-0.063-3.631,1.385-3.634,3.74c-0.002,2.184,1.187,3.583,3.36,3.646 C57.239,13.96,58.395,12.481,58.542,9.978z"/> <path fill-rule="evenodd" clip-rule="evenodd" fill="#FFCF46" d="M81.444,61.483c-4.397,0-8.795-0.039-13.191,0.031 c-0.99,0.012-2.651-0.614-2.703,1.079c-0.058,1.92,1.724,1.42,2.857,1.432c4.396,0.055,8.796,0.101,13.191-0.021 c2.258-0.063,3.18,0.37,3.349,3.033c0.764,12.053-4.467,13.096-14.114,15.288c-5.535,1.258-11.469,0.591-16.927-1.854 c-4.513-2.019-7.282-4.928-6.933-10.376c0.346-5.377,0.405-10.823-0.009-16.195c-0.498-6.483,3.535-11.252,11.379-11.254 c5.277-0.001,10.59-0.403,15.822,0.078c7.481,0.686,13.346-5.798,13.604-13.338c0.009-0.25-0.055-0.519,0.01-0.75 c0.659-2.318-1.686-5.762,1.452-6.933c4.741-1.768,10.087,0.091,12.51,3.688c3.053,4.53,3.466,9.861,3.959,15.066 c0.534,5.61-1.039,10.891-3.396,15.952c-1.601,3.433-4.293,5.092-8.043,5.096c-4.271,0.003-8.542,0-12.816,0 C81.444,61.497,81.444,61.49,81.444,61.483z M80.299,72.68c-0.47-2.142-1.436-3.604-3.924-3.665 c-2.588-0.062-3.256,1.874-3.243,3.602c0.015,1.837,0.527,4.098,3.328,3.886C78.789,76.325,79.925,74.883,80.299,72.68z"/> <path fill-rule="evenodd" clip-rule="evenodd" fill="#386F9F" d="M12.292,116.722c0,2.677,0.121,5.184-0.057,7.669 c-0.077,1.073,1.105,3.09-1.144,3.088c-1.982,0-2.491-1.578-2.428-3.485c0.121-3.639,0.035-7.285,0.035-10.927 c0-2.765,0-5.529,0-8.293c0-4.998,1.341-6.69,6.121-7.724c4.444-0.967,7.936,0.529,9.732,4.419 c1.795,3.891,1.934,7.901,0.137,11.859c-1.77,3.901-5.032,5.357-9.211,4.201C14.525,117.269,13.564,117.044,12.292,116.722z M12.292,106.926c0,1.379,0.187,2.785-0.038,4.127c-0.521,3.101,1.459,3.865,3.794,4.417c2.31,0.545,3.893-0.437,4.819-2.471 c1.556-3.411,1.817-6.929,0.635-10.537c-1.049-3.206-3.472-3.581-6.301-3.182c-2.793,0.396-3.171,2.296-2.928,4.644 C12.378,104.914,12.292,105.925,12.292,106.926z"/> <path fill-rule="evenodd" clip-rule="evenodd" fill="#386F9F" d="M42.347,116.556c-1.913,0.497-3.574,1.029-5.275,1.347 c-3.721,0.697-6.478-1.347-7.447-5.073c-1.203-4.612-0.185-9.206-0.154-13.808c0.014-2.097,1.955-1.279,3.298-2.403 c0,4.925,0.018,9.176-0.007,13.424c-0.013,2.133,0.538,3.888,2.556,5.013c2.227,1.242,4.054,0.059,5.993-0.738 c1.39-0.572,1.216-1.738,1.218-2.848c0.011-4.014-0.001-8.025,0.013-12.039c0.002-0.99-0.101-2.201,1.361-2.152 c1.303,0.043,1.871,0.817,1.849,2.238c-0.095,6.643,0.093,13.295-0.177,19.929c-0.204,5.016-3.742,8.08-8.834,8.305 c-0.761,0.032-1.56,0.102-1.845-0.843c-0.434-1.456,0.744-1.181,1.519-1.429C42.015,123.698,42.747,122.634,42.347,116.556z"/> <path fill-rule="evenodd" clip-rule="evenodd" fill="#386F9F" d="M66.012,99.319c1.481-0.7,2.332-1.341,3.269-1.515 c5.253-0.971,9.279-0.601,9.268,6.476c-0.006,3.644,0.003,7.29-0.006,10.933c-0.002,1.021,0.063,2.014-1.458,2.102 c-1.604,0.095-1.957-0.822-1.949-2.08c0.018-3.512,0.113-7.023,0.111-10.535c-0.003-2.726-0.559-5.14-4.027-5.131 c-3.043,0.005-5.197,2.275-5.213,5.409c-0.017,3.391-0.017,6.784,0.003,10.175c0.004,1.205,0.029,2.262-1.712,2.162 c-1.569-0.094-1.376-1.15-1.376-2.124c-0.011-8.918,0.033-17.843-0.043-26.766c-0.013-1.564,0.604-2.534,1.991-2.601 c1.957-0.09,1.061,1.622,1.09,2.48C66.087,91.8,66.012,95.304,66.012,99.319z"/> <path fill-rule="evenodd" clip-rule="evenodd" fill="#386F9F" d="M82.591,107.314c-0.199-6.963,3.104-10.68,9.083-10.464 c3,0.11,5.251,1.442,6.703,4.033c2.566,4.585,2.219,9.172-0.558,13.466c-1.907,2.948-4.841,4.339-8.418,3.58 c-3.528-0.747-6.29-3.923-6.79-7.624C82.478,109.325,82.591,108.31,82.591,107.314z M96.401,107.477 c-0.158-1.216-0.129-2.91-0.635-4.422c-0.692-2.074-1.953-3.743-4.525-3.856c-2.595-0.113-3.89,1.404-4.756,3.504 c-1.36,3.296-1.271,6.606,0.201,9.843c0.946,2.085,2.406,3.611,4.936,3.188c2.491-0.416,3.961-2.049,4.47-4.472 C96.323,110.164,96.286,109.014,96.401,107.477z"/> <path fill-rule="evenodd" clip-rule="evenodd" fill="#386F9F" d="M103.666,108.404c0-1.256-0.024-2.511,0.004-3.761 c0.058-2.411-0.71-5.082,2.54-6.313c3.722-1.406,7.412-2.46,11.159-0.284c1.402,0.815,1.918,2.296,1.933,3.868 c0.038,4.387-0.02,8.776,0.039,13.162c0.021,1.531-0.381,2.296-2.102,2.255c-1.885-0.045-1.487-1.331-1.475-2.357 c0.045-3.504,0.288-7.006,0.222-10.505c-0.087-4.406-2.505-6.204-6.624-5.122c-1.709,0.45-2.777,1.279-2.693,3.394 c0.167,4.129,0.038,8.272,0.054,12.408c0.004,1.199,0.022,2.26-1.708,2.16c-1.584-0.091-1.325-1.187-1.337-2.138 C103.649,112.915,103.669,110.658,103.666,108.404z"/> <path fill-rule="evenodd" clip-rule="evenodd" fill="#386F9F" d="M58.493,98.633c-0.258,0.307-0.476,0.788-0.704,0.792 c-4.111,0.058-2.97,3.125-3.076,5.401c-0.1,2.121,0.001,4.251-0.03,6.378c-0.031,2.113,0.152,4.077,2.724,4.638 c0.737,0.159,1.073,0.584,0.919,1.321c-0.169,0.812-0.817,0.78-1.458,0.772c-2.956-0.052-5.146-2.141-5.233-5.2 c-0.089-3.125,0.032-6.257-0.042-9.379c-0.038-1.482,0.732-3.375-1.659-3.963c-0.262-0.063-0.355-0.806-0.526-1.23 c3.189-1.001,1.865-3.752,2.303-5.839c0.208-0.988,1.146-1.259,2.033-1.342c0.897-0.085,0.904,0.64,0.933,1.247 c0.118,2.406-1.127,5.625,3.301,5.449C58.126,97.673,58.299,98.261,58.493,98.633z"/> <path fill-rule="evenodd" clip-rule="evenodd" fill="#FEFEFE" d="M58.542,9.978c-0.147,2.503-1.303,3.982-3.691,3.913 c-2.173-0.063-3.363-1.461-3.36-3.646c0.003-2.355,1.24-3.803,3.634-3.74C57.313,6.564,58.374,8.005,58.542,9.978z"/> <path fill-rule="evenodd" clip-rule="evenodd" fill="#FEFEFE" d="M80.299,72.68c-0.374,2.203-1.51,3.646-3.839,3.822 c-2.801,0.212-3.313-2.049-3.328-3.886c-0.013-1.728,0.655-3.663,3.243-3.602C78.863,69.076,79.829,70.538,80.299,72.68z"/> <path fill-rule="evenodd" clip-rule="evenodd" fill="#FEFEFE" d="M12.292,106.926c0-1.001,0.086-2.012-0.018-3.002 c-0.244-2.348,0.134-4.248,2.928-4.644c2.829-0.399,5.252-0.024,6.301,3.182c1.182,3.608,0.921,7.126-0.635,10.537 c-0.926,2.034-2.509,3.016-4.819,2.471c-2.335-0.552-4.314-1.316-3.794-4.417C12.479,109.711,12.292,108.305,12.292,106.926z"/> <path fill-rule="evenodd" clip-rule="evenodd" fill="#FEFEFE" d="M96.401,107.477c-0.115,1.537-0.078,2.688-0.31,3.785 c-0.509,2.423-1.979,4.056-4.47,4.472c-2.529,0.423-3.989-1.104-4.936-3.188c-1.472-3.236-1.562-6.547-0.201-9.843 c0.866-2.1,2.161-3.617,4.756-3.504c2.572,0.113,3.833,1.782,4.525,3.856C96.272,104.566,96.243,106.261,96.401,107.477z"/> </g> </svg>
```

**Review by amacado — CHANGES_REQUESTED** [review]

**Comment by Altijd-youri** [comment]
The python foundation has two svg of their logo on the website. Why not strip their svg and add the word 'python' below?

Python logo from Python foundation
From https://www.python.org/community/logos/

**Review by Thomas-Boi — APPROVED** [review]
While the file is not optimized since the bot is failing for some reason (can't checkout repo), I'll accept the PR and optimize it manually.

**Comment by Thomas-Boi** [comment]
Thanks @Kartstig for fixing this. Sorry for taking a long time to get around to this PR. Life has been busy 😅

**Pull request merged**